code/game/objects/items/weapons/defib.dm

```diff
@@ -109,7 +109,8 @@
 	target.visible_message("<span class='danger'>[target] has been shocked in the chest with the [src] by [user]!</span>")
 	target.Weaken(rand(6,12))
 	target.apply_damage(rand(30,60),BURN,"chest")
-	heart.damage += rand(5,60)
+	if(!(heart.modifiers & OMODIFER_UBER))
```

Comment:
You have totally missed the point of the apply damage proc, put the uber code there instead of spaghettifying it